apps/artemis-web-e2e/src/integration/app2.spec.ts

```diff
@@ -3,12 +3,41 @@ describe('artemis-web', () => {
     Cypress.Cookies.preserveOnce('remember_me', 'sid', 'access_token');
   });
 
-  it('logs in with ldap', () => {
+  it('[LDAP] admin logs in', () => {
     cy.visit('/login');
     cy.get('h1').should('have.text', 'Sign In');
     cy.typeLogin({ email: 'hermes@planetexpress.com', password: 'hermes' });
     cy.loginLDAP();
     cy.waitFor('h1');
     cy.get('h1').should('have.text', 'Dashboard');
   });
+
+  it('[LDAP] admin visits System page', () => {
+    cy.visit('/admin/system');
+    cy.waitFor('h1');
+    cy.get('h1').should('have.text', 'System');
+  });
+
+  it('[LDAP] admin logs out', () => {
+    cy.visit('/dashboard');
+    cy.get('#logout', { timeout: 2000 }).click({ force: true });
+    cy.wait(2000);
+    cy.get('h1').should('have.text', 'Sign In');
+  });
+
+  it('[LDAP] user logs in', () => {
+    cy.visit('/login');
+    cy.get('h1').should('have.text', 'Sign In');
+    cy.typeLogin({ email: 'fry@planetexpress.com', password: 'fry' });
+    cy.loginLDAP();
+    cy.waitFor('h1');
+    cy.get('h1').should('have.text', 'Dashboard');
+  });
+
```

Comment:
can you add a check here that verifies the user is not admin?

Reply:
I was thinking about this but I do not have enough exp with cypress. E.g., is there a way to encode that when I visit the `/admin/system` page as a user I should get a 401 unauthorized? If yes, let me know please, since this would of course be helpful.